json-lint/schemas/inventory/adservers/adthrive.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
This json schema does not match AMP: https://github.com/ampproject/amphtml/blob/master/ads/adthrive.md

We need to respect AMP format, and if we need extra params for Touch, as @ettoreciprianmrf says we can specify them inside the `json` attribute